module/app/app.go

```diff
@@ -261,24 +265,25 @@ func NewGravityApp(
 	legacyAmino := encodingConfig.Amino
 	interfaceRegistry := encodingConfig.InterfaceRegistry
 
-	bApp := baseapp.NewBaseApp(appName, logger, db, encodingConfig.TxConfig.TxDecoder(), baseAppOptions...)
+	bApp := *baseapp.NewBaseApp(appName, logger, db, encodingConfig.TxConfig.TxDecoder(), baseAppOptions...)
```

Comment:
For any New___() functions that return pointers I assigned a variable to the base value, then referenced the base value.